lib/Logging.js

```diff
@@ -0,0 +1,149 @@
+const winston = require("winston");
+require('winston-daily-rotate-file');
+
+class Logging {
+    constructor() {
+        this.loggers = new Map();
+        this.transports = null;
+    }
+
+    /*
+        config:
+            console: "error|warn|info|debug|off"
+            files: {
+                "abc.log" => "error|warn|info|debug|off"
+            }
+            maxFiles: 5
+    */
+    Configure(config) {
+        let updatingLoggers = false;
+        const timestampFn = function() {
+            return new Date().toISOString().replace(/T/, ' ').replace(/\..+/, '');
+        };
+        const formatterFn = function(opts) {
+            return opts.timestamp() + ' ' +
+            opts.level.toUpperCase() + ':' +
+            (opts.meta && opts.meta.loggerName ? opts.meta.loggerName : "") + ' ' +
+            (undefined !== opts.message ? opts.message : '');
+        };
+        // if (this.transports.length > 0) {
+        //     updatingLoggers = true;
+        //     this.transports.forEach((transport) => {
+        //         this.loggers.forEach((wrapper, name) => {
+        //             wrapper.logger.remove(transport);
+        //         });
+        //     });
+        // }
+        this.transports = [];
+        if (config.console != undefined && config.console != "off") {
```

Comment:
Generally best to use `!==` unless explicitly useful to do otherwise. ` a != undefined` evaluates `true` if `a = null` or `a = undefined`. Likewise for the second comparison.